### ISSUE TITLE
operationId is not generated

### ISSUE THREAD

**Issue by fazpu** [kminami/apib2swagger#43]
I would expect the generated code contains something like this:
```
"/product/get-product-list/": {
      "get": {
        "operationId": "product_get-product-list_list",
        "description": "",
        "parameters": [...]
```

However, `operationId` is not being generated. This causes issues with some front-end generators based on swagger.

**Comment by alexkreidler** [comment]
This is fixed in #44 and also in my [fork](https://github.com/alexkreidler/apib2swagger). Hopefully, the maintainer can take another look at these.

**Comment by kminami** [comment]
OK, I'll take a look.

**Comment by kminami** [comment]
I have adopted @alexkreidler 's fork and released as v1.11.0.
Thank you!
